tsl/test/expected/compression_create_chunks.out

```diff
@@ -0,0 +1,74 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+\c :TEST_DBNAME :ROLE_SUPERUSER
+CREATE TABLE conditions (
+    "time" TIMESTAMPTZ NOT NULL,
+    city TEXT NOT NULL,
+    temperature INTEGER NOT NULL,
+    device_id INTEGER NOT NULL
+);
+SELECT table_name FROM create_hypertable('conditions', 'time');
+ table_name 
+------------
+ conditions
+(1 row)
+
+INSERT INTO
+    conditions ("time", city, temperature, device_id)
+VALUES
+  ('2021-06-14 00:00:00-00', 'Moscow', 26,1),
+  ('2021-06-15 00:00:00-00', 'Berlin', 22,2),
+  ('2021-06-16 00:00:00-00', 'Stockholm', 24,3),
+  ('2021-06-17 00:00:00-00', 'London', 24,4),
+  ('2021-06-18 00:00:00-00', 'London', 27,4),
+  ('2021-06-19 00:00:00-00', 'Moscow', 28,4);
+ALTER TABLE conditions SET (timescaledb.compress);
+SELECT * FROM _timescaledb_catalog.chunk;
```

Comment:
Please add ORDER BY 1 ,2;
Sometimes the order of rows changes, we can cause the test to fail randomly.